File: src/main/java/plugin/command/administrator/SetPrayerBookCommandPlugin.java
package plugin.command.administrator;

import org.redrune.game.entity.actor.player.Player;
import org.redrune.game.content.plugin.type.CommandPlugin;
import plugin.command.CommandManifest;

/**
 * @author Tyluur <itstyluur@icloud.com>
 * @since 9/8/2017
 */
@CommandManifest(description = "Sets your prayer book [1/2]", types = { Integer.class })
public class SetPrayerBookCommandPlugin extends CommandPlugin {
	
	@Override
	public void handle(Player player, String[] args, boolean console, boolean clientCommand) {
		int bookId = intParam(args, 1);
		player.getPrayer().setPrayerBook(bookId == 2);
	}
	
	@Override
	public String[] identifiers() {
		return arguments("setprayerbook");
	}
}

File: src/main/kotlin/plugin/command/player/SetPrayerBookCommandPlugin.kt
package plugin.command.player

import plugin.command.CommandManifest
import org.redrune.game.content.plugin.type.CommandPlugin
import org.redrune.game.global.World
import org.redrune.utility.constants.SkillConstants
import plugin.command.player.YellCommandPlugin
import org.redrune.utility.functions.Misc
import org.redrune.game.entity.actor.player.data.PlayerRight
import org.redrune.utility.constants.InterfaceConstants
import org.redrune.cache.loaders.ItemDefinitions
import org.redrune.game.entity.actor.player.Player

/**
 * @author Tyluur <itstyluur></itstyluur>@icloud.com>
 * @since 9/8/2017
 */
@CommandManifest(description = "Sets your prayer book [1/2]", types = [Int::class])
class SetPrayerBookCommandPlugin : CommandPlugin() {
    override fun handle(player: Player, args: Array<String>, console: Boolean, clientCommand: Boolean) {
        val bookId = intParam(args, 1)
        player.prayer.setPrayerBook(bookId == 2)
    }

    override fun identifiers(): Array<String> {
        return arguments("setprayerbook")
    }
}